client/my-sites/themes/theme-showcase.jsx

```diff
@@ -526,6 +530,9 @@ class ThemeShowcase extends Component {
 					title={ this.props.analyticsPageTitle }
 					properties={ { is_logged_in: isLoggedIn } }
 				/>
+				{ siteId && isJetpackSite && isPossibleJetpackConnectionProblem && (
+					<JetpackConnectionHealthBanner siteId={ siteId } />
+				) }
```

Comment:
Would it be possible to render the banner inside [the `SingleSiteJetpack` component](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/themes/single-site-jetpack.jsx) so we don't need the `siteId && isJetpackSite` check?

Reply:
@mmtr Yep! Moved in 5097dd76ee